sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -295,28 +295,48 @@ class GrpcClientDataChannelFactory(DataChannelFactory):
   Caches the created channels by ``data descriptor url``.
   """
 
-  def __init__(self):
+  def __init__(self, credentials=None):
     self._data_channel_cache = {}
     self._lock = threading.Lock()
+    self._credentials = None
+    if credentials is not None:
+      logging.info('Creating secure channel creds.')
```

Comment:
You're not creating them here, are you? If we want to log something, log the ones we're using.

Reply:
done.